dayu_widgets/menu.py

```diff
@@ -541,18 +541,34 @@ def set_value(self, data):
         self.setProperty("value", data)
 
     def _set_value(self, value):
-        data_list = value if isinstance(value, list) else [value]
+        """
+        mode 1: Cascader
+        mode 2: Exclusive
+        mode 3: Single
+        """
+        if self.property("cascader"):
+            selected_data = ['/'.join(map(str, value))]
+        else:
+            if self._action_group.isExclusive():
+                selected_data = [value]
+            else:
+                selected_data = value
```

Comment:
你看看如果是修改成这样是否也能符合预期？
```suggestion
            selected_data = ['/'.join(map(str, value))]
        else:
            selected_data = [value]
```

Reply:
把 "/" 改为 self.property("separator") 吧